docs/content/preview/explore/fault-tolerance/macos.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: High availability during failures (YugabyteDB Anywhere)
+title: High availability during failures in a local universe
 headerTitle: High availability during node and zone failures
 linkTitle: HA during failures
-description: Simulate fault tolerance and resilience in a local YugabyteDB database universe.
```

Comment:
I would suggest you retain "YugabyteDB database"- this helps with SEO as suggested in the past.